openstack_dashboard/test/selenium/integration/test_user_credentials.py

```
@@ -0,0 +1,175 @@
+    rows = driver.find_elements_by_css_selector(
+        f"table#credentialstable tr[data-display='{user_credential_blob}']")
+    assert len(rows) == 1
+    rows[0].find_element_by_css_selector(".data-table-action").click()
+    user_credential_form = driver.find_element_by_css_selector(
+        ".modal-dialog form")
+    user_credential_form.find_element_by_id("id_data").clear()
+    user_credential_form.find_element_by_id("id_data").send_keys(
+        f"EDITED_{user_credential_blob}")
+    user_credential_form.find_element_by_css_selector(".btn-primary").click()
+    messages = widgets.get_and_dismiss_messages(driver)
+    assert(f"Success: User credential updated successfully." in messages)
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)